markdown_lab/core/scraper.py

```
@@ -850,45 +743,107 @@
             try:
-                self._process_single_url(
-                    url, i, len(links), output_path, output_format,
-                    save_chunks, chunk_directory, chunk_format
-                )
-                successfully_scraped.append(url)
-            except Exception as e:
-                logger.error(f"Error processing URL {url}: {e}")
-                continue
-
+                import concurrent.futures
+                
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)